aa-sdk/core/src/actions/smartAccount/waitForUserOperationTransacation.ts

```diff
@@ -5,6 +5,7 @@ import { IncompatibleClientError } from "../../errors/client.js";
 import { FailedToFindTransactionError } from "../../errors/transaction.js";
 import { Logger } from "../../logger.js";
 import type { WaitForUserOperationTxParameters } from "./types.js";
+import {_waitForUserOperationTransaction} from "./internal/waitForUserOperation";
```

Comment:
🚫 **[eslint]** <[import/extensions](https://github.com/import-js/eslint-plugin-import/blob/v2.29.0/docs/rules/extensions.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing file extension for "./internal/waitForUserOperation"
<!-- __reviewdog__:ChBhNjJlYzk5MWQ1M2M4OWQ3EgZyZGpzb24= -->